.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This file should be gitignored, on line 23 of the git ignore it seems like it didn't ignore it properly?

Reply:
that's as expected,